pkgs/development/python-modules/dipy/default.nix

```diff
@@ -0,0 +1,83 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, isPy27
+, packaging
+, pytest
+, python
+, cython
+, numpy
+, scipy
+, pandas
+, scikitlearn
+, statsmodels
+, h5py
+, tables
+, cvxpy
+, matplotlib
+, nibabel
+, withCvxpy ? false  # unfree dependency on `mkl`
+}:
+
+buildPythonPackage rec {
+  pname = "dipy";
+  version = "1.1.1";
+
+  disabled = isPy27;
+
+  src = fetchFromGitHub {
+    owner  = "nipy";
+    repo   = pname;
+    rev    = version;  # "1fb90ab8bbec66bdf26c59cc04a90df3cfc88510";
+    #sha256 = "08sfxb45fvf8g165skwq1ziig98h562jiqp49855yhgv2m5l3icg";
+    sha256 = "08abx0f4li6ya62ilc59miw4mk6wndizahyylxhgcrpacb6ydw28";
+  };
+
+  buildInputs = [ cython ];
+  propagatedBuildInputs = [
+    packaging
+    numpy
+    scipy
+    pandas
+    h5py
+    nibabel
+    tables
+    matplotlib
+    scikitlearn
+    statsmodels
```

Comment:
these seem to be extras or checkInputs, generally, these are not propagated by default

Reply:
Extras mostly, now removed.